pulp_rpm/app/migrations/0029_limit_checksum_choices.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.17 on 2020-11-26 14:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rpm', '0028_rpmrepository_last_sync_repomd_cheksum'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='package',
+            name='checksum_type',
+            field=models.CharField(choices=[('md5', 'md5'), ('sha1', 'sha1'), ('sha224', 'sha224'), ('sha256', 'sha256'), ('sha384', 'sha384'), ('sha512', 'sha512')], max_length=10),
```

Comment:
Before this change, more checksum types were allowed.
Shouldn't we handle it for the removed types here? E.g. UNKNOWN and SHA?